### ISSUE TITLE
Can't play audio recorded on Android

### ISSUE THREAD

**Issue by DevilWarrior** [ZaraclaJ/audio_recorder#20]
Hi, this plugins works for me except when a file was recorded on an Android and than played on an iPhone. I can record/play within an iPhone, I can record/play within android. I can reproduce in an Android audio recorded on an iPhone but I can't reproduce on an iPhone audio recorder on an Android. I hope the tong twist makes sense

I tried both playing the file straight from http and also downloading it locally and playing it afterwards.

Any ideas?

PS: My code is pretty vanilla from the example hence why I didn't include it

**Comment by ZaraclaJ** [comment]
Have you checked if the audio format of your file created on Android is playable on iPhone ?

**Comment by ferbass** [comment]
Same issue here the mine-type for the android file is video/mp4

```
android -> 02ad9210-bd30-11e8-80ae-094b53e3c499.m4a:                    video/mp4
iOS ->         26adba00-bd30-11e8-f6be-63cd3749b580.m4a:                    audio/x-m4a
```

**Comment by ferbass** [comment]
@DevilWarrior @ZaraclaJ I opened a Pull Request to fix this problem https://github.com/ZaraclaJ/audio_recorder/pull/26

**Comment by ZaraclaJ** [comment]
Fixed in the 1.0.0 version of the plugin
